website/content/docs/secrets/ssh/signed-ssh-certificates.mdx

```diff
@@ -466,6 +466,50 @@ forwarding. See [no prompt after login](#no-prompt-after-login) for examples.
 }
 ```
 
+### Key Comments
+There are additional steps needed to preserve [comment attributes](https://docs.oasis-open.org/kmip/kmip-spec/v2.1/cs01/kmip-spec-v2.1-cs01.html#_Toc32239331)
+in keys which ought to be considered if they are required. Private and public 
+key may have comments applied to them and for example where `ssh-keygen` is used 
+with its `-C` parameter - similar to:
+
+```shell-session
+ssh-keygen -C "...Comments" -N "" -t rsa -b 4096 -f host-ca
+```
+
+Adapted key values containing comments must be provided with the key related 
+parameters as per the Vault CLI and API steps demonstrated below. 
+
+```shell-extension
+# Using CLI:
+vault secrets enable -path=hosts-ca ssh
+KEY_PRI=$(cat ~/.ssh/id_rsa | tr '\n' '*' | sed 's/\*/\\n/g')
+KEY_PUB=$(cat ~/.ssh/id_rsa.pub | tr '\n' '*' | sed 's/\*/\\n/g')
+# Create / Update keypair in Vault
+vault write ssh-client-signer/config/ca \
+  generate_signing_key=false \
+  private_key="${KEY_PRI}" \
+  public_key="${KEY_PUB}"
+```
+
+```shell-extension
+# Using API:
+curl -X POST -H "X-Vault-Token: ..." -d '{"type":"ssh"}' http://127.0.0.1:8200/v1/sys/mounts/hosts-ca
+KEY_PRI=$(cat ~/.ssh/id_rsa | tr '\n' '*' | sed 's/\*/\\n/g')
```

Comment:
Would `sed -z 's/\n/\\n/g'` be cleaner than the dual `tr+sed` pipe? Or is `-z` not portable?

Reply:
`-z` is portable at least going by all POSIX variance of sed.